src/bootstrap/native.rs

```diff
@@ -178,7 +178,7 @@ impl Step for Llvm {
             .define("LLVM_TARGET_ARCH", target_native.split('-').next().unwrap())
             .define("LLVM_DEFAULT_TARGET_TRIPLE", target_native);
 
-        if !target.contains("netbsd") {
+        if !target.contains("netbsd") && target != "aarch64-apple-darwin" {
```

Comment:
IMO such special exceptions should always come with a comment saying why they are needed.

Reply:
Doesn't hurt, though in this case the code changes infrequently and git blame is pretty good.